python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,233 @@ class TabConfigView(TabDefault):
+        label_power_motor = create_label(
+            name="Motor", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_power_motor)
+        layout.addWidget(self._label_power_warning_motor)
+        layout.addWidget(self._label_power_fault_motor)
+        layout.addWidget(self._label_power_threshold_motor)
+
+        label_power_communication = create_label(
+            name="Communication", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_power_communication)
+        layout.addWidget(self._label_power_warning_communication)
+        layout.addWidget(self._label_power_fault_communication)
+        layout.addWidget(self._label_power_threshold_communication)
+
+        layout.addSpacing(space_between_sections)
+
+        label_in_position_flag = create_label(
+            name="In-Position Flag", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_in_position_flag)
+        layout.addWidget(self._label_in_position_axial)
+        layout.addWidget(self._label_in_position_tangent)
+        layout.addWidget(self._label_in_position_sample)
+
+        layout.addSpacing(space_between_sections)
+
+        label_communication_timeout = create_label(
+            name="Communication Timeout", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_communication_timeout)
+        layout.addWidget(self._label_timeout_sal)
+        layout.addWidget(self._label_timeout_crio)
+        layout.addWidget(self._label_timeout_ilc)
+
+        layout.addSpacing(space_between_sections)
+
+        label_miscellaneous = create_label(
+            name="Miscellaneous", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_miscellaneous)
+
+        label_inclinometer = create_label(
+            name="Inclinometer", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_inclinometer)
+        layout.addWidget(self._label_misc_range_angle)
+        layout.addWidget(self._label_misc_diff_enabled)
+
+        label_temperature = create_label(
+            name="Temperature", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_temperature)
+        layout.addWidget(self._label_misc_range_temperature)
+
+        return layout
+
+    def _set_signal_config(self, signal_config):
+        """Set the config signal with callback function.
+
+        Parameters
+        ----------
+        signal_config : `SignalConfig`
+            Signal of the configuration.
+        """
+
+        signal_config.config.connect(self._callback_signal_config)
+
+    @Slot()
+    def _callback_signal_config(self, config):
+        """Callback of the config signal for the new configuration.
+
+        Parameters
+        ----------
+        config : `Config`
+            New configuration.
+        """
+
+        self._label_file_configuration.setText(
```

Comment:
See above. Storing keys in hash will allow you to rewrite that to a loop, using getattr(config, name) to retrieve configuration item (in case you don't switch for configuration to names - remember, in Python accessing class variable is roughly same as accessing a key in a hash).

Reply:
Fixed in the commit: [29ca788](https://github.com/lsst-ts/ts_m2gui/pull/6/commits/29ca788f3c2b709d92963b35534482e61bb71a11). Thanks!